arlpy/bf.py

```diff
@@ -271,7 +272,69 @@ def capon(x, fc, sd, complex_output=False):
             R += _np.random.normal(0, _np.max(_np.abs(R))/1000000, R.shape)
         return _np.array([1.0/a[j].conj().dot(_np.linalg.inv(R)).dot(a[j]).real for j in range(a.shape[0])])
 
-def broadband(x, fs, nfft, sd, f0=0, fmin=None, fmax=None, overlap=0, beamformer=bartlett):
+def music(x, fc, sd, complex_output=False,num_sig_components = 1):
```

Comment:
Stylistic suggestion: `False,num_sig_components = 1` => `False, nsignals=1`
(whitespace and parameter naming consistent with the rest of the code)